js/ui/components/clubhouse.js

```diff
@@ -72,6 +72,21 @@ const ClubhouseIface =
   </interface> \
 </node>';
 
+function _clipToMonitor(actor) {
```

Comment:
My gut feeling is that this would be better implemented as a ClutterConstraint applied to the actor.